### PR TITLE
Pre-staking finalized. Merge for 11th nov

### PR DESCRIPTION
**Pull request by onmax**
Updated content to reflect end of pre-staking period

### CODE REVIEW

File: app/content/prestaking.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export default {
   hero: {
-    title: 'Pre-stake to earn your share of 200 Million NIM in rewards and bonus giveaways!',
-    body: 'Join in and support the network. Be among the first to earn, get a unique Identicon upgrade and win some NIM.',
+    title: 'Pre-staking is closed. Get Ready for What’s Next!',
+    body: 'Thank you to everyone who joined in and supported the network. If you pre-staked, thanks! You ll receive your rewards directly in your wallet within the next 30 days.',
```
Comment (sisou) [inline app/content/prestaking.js:4]:
```suggestion
    body: 'Thank you to everyone who joined in and supported the network. If you pre-staked, thanks! You'll receive your rewards directly in your wallet within the next 30 days.',
```

Comment (sisou) [reply app/content/prestaking.js:4]: And I thought it's 3 months, not 30 days?

File: app/content/prestaking.js [hunk 2]
```diff
@@ -30,8 +30,8 @@ export default {
 
   communityGoals: {
     label: 'Community',
-    title: 'Join in and help secure the network',
-    body: 'The more people pre-stake, the more secure the young network will be. Join in, help reach the base goal and let’s smash the stretch goal right after.',
+    title: 'Get ready to start staking NIM!',
+    body: 'The official Proof of Stake launch begins November 19th. Earn rewards, support the network, and be part of the new Nimiq era. Help secure the network and start earning!',
```
Comment (sisou) [inline app/content/prestaking.js:34]: We are not using the word 'launch'.

**Comment by onmax** [comment]
Fyi, this PR is not finished as there are some missing things.

For now I just copied pasted without reviewing it much

File: app/content/prestaking.js (hunk 2 above)
Comment (martindesouz) [reply app/content/prestaking.js:34]: Correct. Should be "migration"

File: app/content/prestaking.js (hunk 1 above)
Comment (martindesouz) [reply app/content/prestaking.js:4]: I think it is 3-months. Yes. I had written (insert proper duration) in the notes. Thanks.

**Pull request merged**